flambe/optim/noam.py

```diff
@@ -53,4 +53,4 @@ def lr_lambda(self, step: int) -> float:
             if step > self.warmup:
                 return 1. / (self.d_model ** 0.5) / (step ** 0.5)
             else:
-                return step / (self.d_model ** 0.5) / (self.warmup ** 1.5)
+                return (step + 1) / (self.d_model ** 0.5) / (self.warmup ** 1.5)
```

Comment:
i took the formula from the original implementation. Not sure there should be a one here

Reply:
Yes. But that sets the learning rate to zero when the step equals zero (which I think I've seen happen). Given that we also have `step > warmup` instead of `step >= warmup`, I think that this may be more consistent. I'm happy to change it back, though - it does not really matter that much.